### DISCUSSION TITLE
Subtitle Edit v4.0.5 with Batch Audio to Text Purfview's Faster Whisper

### DISCUSSION THREAD

**Discussion by Fetterbr61** [SubtitleEdit/subtitleedit#8286] [General]
Hi.
I am new to this program but i like it until now :) Question. I am using the program Subtitle Edit v4.0.5 with Batch Audio to Text Purfview's Faster Whisper, where can I add the Fix common errors or do i have to reload every single movie again to do this ?

**Comment by Gardengul** [comment]
You can use batch mode for this.

**Comment by Fetterbr61** [comment]
How ? Have you tryed it

**Reply by niksedk** [reply]
> How ? Have you tryed it

Try Tools - Batch convert, and select "Fix common errors"

**Reply by Fetterbr61** [reply]
you haven't understood, i want to Batch Audio to Text Purfview's Faster Whisper and not convert.

**Reply by Fetterbr61** [reply]
but now when we are about convert, i have actually payed Libre Translate but i never fund out how to get it to work, itys the Auto translate i click
